docs/VLCB4in4out_README.md

```diff
@@ -114,27 +115,36 @@ The following EV values are defined to control the LEDs in this example:
 
 ### Produced Events
 
-The Events Table in an Uninitiailised module is empty. When the module changes to
+The Events Table in an Uninitialised module is empty. When the module changes to
 Normal Mode from Uninitialised, the Events Table is populated with a default
 producer event for each of the four switches (or, in general, each producer item).
+These default events can be removed by using OPC_EVULN whilst the module is in
+learn mode.
 
 A module can also be taught a short event or, indeed, a spoof event. This is
 easily done using any of the normal FCU teach techniques.  For example, a short
 event can be dragged from the Software Node and dropped onto the VLCB4in4out in
-the Node Window.  When the event variable dialogue opens, put the EV Value into the
-relevant EV box for any LEDs that you wish to use for Consume Own Events (see below)
-and press OK.  Now, within 5 seconds, press the switch that you wish to become the
-short or spoof event 
+the Node Window.  When the event variable dialgue opens, put the index number of
+the switch to be associated with the event in EV1 and press OK.  The selected
+switch will now generate that short event (but see FCU Anomolies below).
+
+If a switch default has been unlearnt and that switch not been assigned to an
+event by the FCU, operation of the switch will result in a new default event
+being generated as long as its NV Value is set for a function other than
+"Do Nothing" (See Node Variables below).
 
 ### Consume Own Events
 
 If the Produced Events Service and the Consume Events Service are both applied,
 the Consume Own Events Service can also be enabled.  This service provides a 
 buffer that will pass the produced event back to the Consumed Event Service.
-A consumed Own Event still only has one entry in the Event Table.  If the Event
-Variables are left as 0 or NULL, then the Consume Events Service will do nothing.
-If the Event Variables are populated as shown in the table in the Consumed Events
-Section above, the LEDs will behave accordingly to a Produced Event.
+A Consume Own Event still only has one entry in the Event Table.  If the Event
+Variables 2 onwards are left as 0 or NULL or any undefined value, then the
```

Comment:
Can an event variable be set to "NULL" ?

Reply:
No, I don't think they can.  I'll update.